src/app/components/core/configuration-manager/configuration-manager.component.html

```diff
@@ -25,12 +25,12 @@
             </div>
           </div>
         </div>
-        <div *ngIf="!isChild" class="panel-block">
-          <button class="button is-rounded is-fullwidth is-small" disabled>
-            no sub-category
-          </button>
+        <div *ngIf="!isChild" class="panel-block no-cat">
+          <div class="has-text-centered">
```

Comment:
`has-text-centered` class is required? we have `justify-content: center;`  on parent div. may be we can remove this div and keep small tag only.

Reply:
fixed!